add did:vaa: method

caict-develop-zhangbo opened the pull request:
Spec here https://github.com/caict-develop-zhangbo/vaa-method/blob/master/README.md


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/caict-develop-zhangbo/did-spec-registries/pull/98.html" title="Last updated on Aug 6, 2020, 4:06 PM UTC (5a4c789)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-spec-registries/98/d3d8a6b...caict-develop-zhangbo:5a4c789.html" title="Last updated on Aug 6, 2020, 4:06 PM UTC (5a4c789)">Diff</a>

OR13 commented:
@caict-develop-zhangbo putting this on your radar: https://github.com/w3c/did-core/issues/174#issuecomment-666414855

caict-develop-zhangbo commented:
> @caict-develop-zhangbo putting this on your radar: [w3c/did-core#174 (comment)](https://github.com/w3c/did-core/issues/174#issuecomment-666414855)

Thanks for review, I've adjusted the "updated', "created" and "revoked" timestamps and added signatures  instructions

caict-develop-zhangbo commented:
> @caict-develop-zhangbo putting this on your radar: [w3c/did-core#174 (comment)](https://github.com/w3c/did-core/issues/174#issuecomment-666414855)

Thanks for review, I've adjusted the "updated', "created" and "revoked" timestamps and added signatures  instructions



> You may want to watch for did core spec changes related to "metadata", this is one of the few did methods I have seen that includes "updated', "created" and "revoked" timestamps in the did document.
> 
> Privacy and Security Considerations section could use a little more clarity, for example:
> 
> "DID documents use signature technology to prevent malicious error correction."
> 
> I don't see signatures on the HTTP based operations, or on the documents themselves.

Thanks for review, I've adjusted the "updated', "created" and "revoked" timestamps and added signatures instructions

msporny commented:
Merge conflicts, please fix.

msporny commented:
Nevermind, just squash-merged.
